sidebars.js

```diff
@@ -323,163 +323,6 @@ const sidebars = {
     },
   ],
 
-  sqlreference: [
```

Comment:
@fernando-aviles,
There's a bit of added complexity here because these docs are in a [separate repo](https://github.com/ClickHouse/ClickHouse/tree/master/docs/en). They get pulled into this repo during the build process using https://github.com/ClickHouse/clickhouse-docs/blob/4698586d236546c55b5e3ea5ae28118d6711c6b6/package.json#L14

The build is failing at the moment because we have a check in place to make sure that all files have a sidebar associated with them to catch "floating pages": https://github.com/ClickHouse/clickhouse-docs/blob/main/plugins/checkFloatingPages.js (runs as part of the Docusaurus build as a plugin)